kubernetes/deployment/out-of-tree/target-cluster-role.yaml

```diff
@@ -50,3 +50,10 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - watch
+  - list
```

Comment:
Just a reminder: We will have to update all the extensions to adapt the cluster-roles to add this post merging of this PR.